### DISCUSSION TITLE
Update User (Client/Server)

### DISCUSSION THREAD

**Discussion by danyhiol** [accounts-js/accounts#1239] [Q&A] [answered]
There is actually no way or function that allows updating the user from the client as well as from the server. Please correct me if I am wrong.
Is there any way to extend `@accounts/graphql-api`   & `@accounts/graphql-client` with new functions such as:

```
async function updateUser(id: string, user: User) => { // Perform update and return user };
accountGraphQLApi['updateUser'] =updateUser; 
accountGraphQLClient['updateUser'] =updateUser;
```
Of course, client and server implementation will be different. 
Or is it better to just make a GraphQL route outside of AccountsJS?

**Comment by pradel** [comment, marked as answer]
Probably better to make a different route outside accounts js that suits your needs

**Reply by danyhiol** [reply]
Thanks for the reply. I will go that way then. But it would be great to have it as an internal feature.
